regression/cbmc-java/stack_var1/stack_var1.j

```diff
@@ -0,0 +1,22 @@
+.class public stack_var1
```

Comment:
ok, will do, the `stack_test.java` is always the entry point and creates an object instance of the class created via jasmin.